inc/deploygroup_staticdata.class.php

```diff
@@ -208,6 +213,69 @@ static function duplicate($source_deploygroups_id, $target_deploygroups_id) {
       }
       return $result;
    }
+
+
+   /**
+    * Form to import computers ID in CSV file
+    *
+    */
+   static function csvimport_form(PluginFusioninventoryDeployGroup $item) {
+
+      echo "<form action='' method='post' enctype='multipart/form-data'>";
+
+      echo "<br>";
+      echo "<table class='tab_cadre_fixe' cellpadding='1' width='600'>";
+      echo "<tr>";
+      echo "<th>";
+      echo __('Import computers list from CSV file (first column must contain the computer ID)', 'fusioninventory')." :";
```

Comment:
The local would be better like this : "Import a list of computers from a CSV file (the first column must contain the computer ID)".